Makefile

```diff
@@ -392,3 +392,12 @@ libsodium:
 		$(MAKE) install; \
 	fi
 .PHONY: libsodium
+
+###############################################################################
+###                                Proto                                    ###
+###############################################################################
+
+proto-swagger-gen:
+	@echo "Generating Protobuf Swagger"
+	./scripts/generate-docs.sh
```

Comment:
finschia-sdk and wasmd runs `generate swagger` in docker (https://github.com/Finschia/finschia-sdk/blob/main/Makefile#L433).
why this does not run in docker?

Reply:
This script need to get the proto files from the go packages. So this environment need golang development environment. But the wasmd and finschia-sdk do not need this progress. And the docker image used by finschia-sdk is very old docker image and installed golang v1.16. So finschia cannot use that docker image. 

So I developed to able to execute without docker image.